apps/ui/src/components/Canvas.tsx

```diff
@@ -454,6 +459,12 @@ function CanvasImpl() {
     }
   }, [escapePressed]);
 
+  useEffect(() => {
+    if (node2children.size == 0) {
+      buildNode2Children();
+    }
+  }, [node2children]);
+
```

Comment:
Move this to here:

https://github.com/codepod-io/codepod/blob/c7a6dd35d7e2d29408bd44a967fdcf02638fee6a/apps/ui/src/lib/store/yjsSlice.ts#L179